parlai/core/image_featurizers.py

```diff
@@ -71,6 +70,7 @@ def init_cnn(self):
         """Lazy initialization of preprocessor model in case we don't need any image preprocessing."""
         try:
             import torch
+            self.torch = torch
```

Comment:
We would need to change line 63 to be `self.use_cuda = False` and then move `self.use_cuda = not opt.get('no_cuda', False) and torch.cuda.is_available()` to this function (or at least after this import) but otherwise looks good

Reply:
oops thanks good  catch